src/xcode/ENA/ENA/Source/AppDelegate & Globals/NotificationManager.swift

```diff
@@ -136,4 +148,17 @@ final class NotificationManager: NSObject, UNUserNotificationCenterDelegate {
 		}
 		return nil
 	}
+	
+	private func findHealthCertifiedPerson(_ identifier: String) -> (HealthCertifiedPerson)? {
+		for person in healthCertificateService.healthCertifiedPersons.value {
+			if let name = person.name?.standardizedName,
+				  let dateOfBirth = person.dateOfBirth {
+				let hashedID = ENAHasher.sha256(name + dateOfBirth)
+				if hashedID == identifier {
+					return person
+				}
+			}
+		}
+		return nil
```

Comment:
did you try to use a `first(where:` instead of the for loop? I also wonder why we need to compare `healthCertifiedPerson` manually. Isn't it equatable?